src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts. In case 'Roboto' is not available, you have 'Arial' and 'Helvetica' as backups, but it's also a good practice to specify a generic font family such as 'sans-serif' as a final fallback.